airflow/www_rbac/templates/airflow/gantt.html

```diff
@@ -47,9 +47,9 @@
 {% block tail %}
 {{ super() }}
 <script src="{{ url_for('appbuilder.static',filename='datepicker/bootstrap-datepicker.js') }}"></script>
-<script src="{{ url_for('static', filename='d3.v3.min.js') }}"></script>
-<script src="{{ url_for('static', filename='d3.tip.v0.6.3.js') }}"></script>/
-<script src="{{ url_for('static', filename='gantt-chart-d3v2.js') }}"></script>
+<script src="{{ url_for_asset('d3.min.js') }}"></script>
+<script src="{{ url_for_asset('d3-tip.js') }}"></script>/
+<script src="{{ url_for_asset('ganttChartD3v2.js') }}"></script>
```

Comment:
Again - why the change in style?

Reply:
https://github.com/apache/incubator-airflow/pull/3572#discussion-diff-200991459R40